test/HazelcastClientTest.js

```diff
@@ -108,7 +108,7 @@ configParams.forEach(function (cfg) {
             expect(info.localAddress.host).to.equal(client.clusterService.getOwnerConnection().localAddress.host);
             expect(info.localAddress.port).to.equal(client.clusterService.getOwnerConnection().localAddress.port);
             expect(info.uuid).to.equal(client.clusterService.uuid);
-            expect(info.type).to.equal('NodeJS');
+            expect(info.type).to.equal('NODEJS');
```

Comment:
breaking change?

Reply:
I changed this because it is mentioned as `NODEJS` in server side.